jwst/tso_photometry/tso_photometry.py

```diff
@@ -79,25 +79,28 @@ def tso_aperture_photometry(datamodel, xcenter, ycenter, radius, radius_inner,
                 'subarray.  No background subtraction was performed.')
 
         for i in np.arange(nimg):
-            aperture_sum.append(np.sum(datamodel.data[i, :, :]))
+            aperture_sum.append(np.nansum(datamodel.data[i, :, :]))
```

Comment:
If there are NaNs here, does that mean the flux will be systematically underestimated?   How is that potential issue handled?

Reply:
That's correct.  For standard imaging modes, the decision was to return NaN in this case for that reason.  For TSO, it sounds like it's preferable to return the values anyway, so there is something to work with, even if it is underestimated.

Reply:
sounds good to me, although it should probably be documented somewhere (maybe it already is?)

Reply:
Let's add an explicit statement of this to readthedocs; I'll do the same on JDox.  Imaging source catalog extractions will return NaN since the most likely failure case is NaN-valued central saturated pixels in an image with many sources, while TSO mode will not return NaN since it's easier for single pixels to be affected in a given integration and science analysis will be focused on variability of one source.